src/style.css

```diff
@@ -0,0 +1,87 @@
+@font-face {
+  font-family: Avenir, sans-serif;
+  src: url(./fonts/Avenir.ttc);
+}
+
+@font-face {
+  font-family: Avenir-Heavy, sans-serif;
+  src: url(./fonts/Avenir-Heavy.ttf);
+}
+
+@font-face {
+  font-family: Avenir-Book, sans-serif;
+  src: url(./fonts/Avenir-Book.ttf);
+}
+
+html {
+  font-size: 16px;
+  font-weight: 300;
+  font-family: Avenir, sans-serif;
+}
+
+.search-bar__wrapper {
+  position: relative;
+  margin: 20px 0;
+}
+
+.search-bar__form {
+  position: relative;
+  display: flex;
+  text-align: center;
+}
+
+.search-bar__label {
+  position: absolute;
+}
+
+.search-bar__label.big {
+  top: 25px;
+  left: 26px;
+  height: 19px;
+  width: 19px;
+}
+
+.search-bar__label.small {
+  top: 12px;
+  left: 14px;
+  height: 11px;
+  width: 11px;
+}
+
+.search-bar__img {
+  height: 100%;
+  width: 100%;
+}
+
+.search-bar__input {
```

Comment:
inputs don't inherit font-family that's why your inputs use Arial

Reply:
Done.